explainers/navigation-to-unsigned-bundles.md

```diff
@@ -0,0 +1,363 @@
+offline use.
+
+### Process of loading a bundle
+
+When the browser navigates to an `application/webbundle;v=___` resource, or
+opens a file that the filesystem identifies as that MIME type, it first parses
+the resource enough to pull out the bundle's [primary
+URL](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#name-load-a-bundles-metadata)
+and a set of flags to control loading (not yet defined in the format). One of
+these flags defines whether the claimed URLs in the bundle are expected to be
+trusted, which for unsigned bundles means the bundle is served from the origin
+that claims it.
+
+### Loading a trusted unsigned bundle.
+
+The browser selects an **unsigned bundle scope** that's based on the bundle's
+URL in the same way the [service worker's scope
+restriction](https://w3c.github.io/ServiceWorker/#path-restriction) is based on
+the URL of the service worker script. If the bundle is served with the
+[`Service-Worker-Allowed`
+header](https://w3c.github.io/ServiceWorker/#service-worker-allowed), that sets
+the unsigned bundle scope to the value of that header. (We are also considering
+using a differently-named header that applies only to bundles.)
+
+Bundles can also contain a
+[manifest](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html#name-parsing-the-manifest-sectio)
+whose [`scope` key](https://www.w3.org/TR/appmanifest/#scope-member) can narrow
+the scope but not broaden it.
+
+> Browsers might also provide a way for device manufacturers to define an area
+  of the local filesystem that can serve trusted unsigned bundles without any
+  scope restriction. This would make it easier to pre-install applications in a way that has a chance of working across different browsers.
+
+The browser redirects to the bundle's primary URL. If that primary URL is within
+the unsigned bundle scope, it also attaches the bundle itself and its unsigned
+bundle scope to the redirect and the subsequent environment settings object.
+That is, the bundle is treated as just a very expensive redirect if it tries to
+claim URL space it's not allowed to define.
+
+Subsequent subresource requests will also only be served from the bundle if
+they're inside its scope.
+
+### Loading an untrusted bundle
+
+The browser redirects to [a URL that refers to the bundle's primary URL *inside
+its bundle*](#urls-for-bundle-components), with the bundle itself attached.
+
+Because the user might save a bundle to a file whose path includes private
+information like their username, APIs like
+[`Window.location`](https://developer.mozilla.org/en-US/docs/Web/API/Window/location)
+and
+[`Document.URL`](https://developer.mozilla.org/en-US/docs/Web/API/Document/URL)
+have to return the claimed URL and not the full encoded URL. Avoiding the new
+scheme in these APIs will also probably improve compatibility.
+
+### Subresource loads with an attached bundle
```

Comment:
Here this sounds like it's excluding navigations while the text below mentions navigations too